sunpy/coordinates/ephemeris.py

```diff
@@ -43,8 +47,25 @@ def get_body_heliographic_stonyhurst(body, time='now'):
     """
     obstime = parse_time(time)
 
-    body_icrs = ICRS(get_body_barycentric(body, obstime))
-    body_hgs = body_icrs.transform_to(HGS(obstime=obstime))
+    if observer is None:
+        body_icrs = get_body_barycentric(body, obstime)
+    else:
+        observer_icrs = SkyCoord(observer).icrs.cartesian
+
+        # This implementation is modeled after Astropy's `_get_apparent_body_position`
+        light_travel_time = 0.*u.s
+        emitted_time = obstime
+        delta_light_travel_time = 1.*u.s # placeholder value
+        while np.any(np.fabs(delta_light_travel_time) > 1.0e-8*u.s):
+            body_icrs = get_body_barycentric(body, emitted_time)
+            distance = (body_icrs - observer_icrs).norm()
+            delta_light_travel_time = light_travel_time - distance / speed_of_light
+            light_travel_time = distance / speed_of_light
+            emitted_time = obstime - light_travel_time
+
+        print('Apparent location factors in {} seconds of light travel time'.format(light_travel_time.to('s').value))
```

Comment:
This should be logger or @ehsteve will flip out :stuck_out_tongue:

Reply:
Too late!!!!

use
```python
from sunpy import log

log.info(f"Apparent location factors in {light_travel_time.to('s')} of light travel time.')
```

Reply:
info? I think we should print this by default?